PocketKit/Sources/PocketKit/Collections/CollectionViewController.swift

```diff
@@ -318,6 +294,134 @@ private extension CollectionViewController {
         }
     }
 
+    /// Determines the stories layout in the native collection view on iPad mode with regular horizontal size class (including split view). Number of columns for the grid layout depends on device orientation and the view's width length. If view's width is less than 800, show 2 col, otherwise show 2 col if it is in portrait and 3 col if it is in landscape mode.
+    /// - Parameters:
+    ///   - stories: list of stories associated with collection to display
+    ///   - width: width that the section occupies
+    ///   - margin: padding adding to the side of the section
+    /// - Returns: section layout for stories on iPad view and regular horizontal size class
+    private func collectionSectionForWideLayout(with stories: [CollectionStory], width: CGFloat, margin: CGFloat) -> NSCollectionLayoutSection {
+        let numberOfColumns: CGFloat = showThreeColumnLayout ? 3 : 2
+        let recommendationsHeight: [CGFloat] = getRecommendationHeights(with: stories, width: width, margin: margin, numberOfColumns: numberOfColumns)
```

Comment:
nit: why not just rely on type inference?

Reply:
removed `[CGFloat]` to rely on type inference

Reply:
Type inference is nearly 2x slower when compiling.
Not difference when running, though.

Reply:
thanks @CMasterson~! I think it was helpful for me at the time of coding to have things explicit, but I'm open to what the team suggests if type inference is more readable. I removed for now